salt/metalk8s/kubeadm/init/certs/etcd-ca.sls

```diff
@@ -38,7 +33,6 @@ Generate etcd CA certificate:
     - require:
       - x509: Create etcd CA private key
 
-# TODO: Find a better way to advertise CA server
 Advertise etcd CA in the mine:
```

Comment:
To be consistent with the previous change for basic CA
```suggestion
Advertise etcd CA certificate in the mine:
```

Reply:
(strange that github suggestion remove a space not existing at the beginning of the line)